### PR TITLE
add metric for mnesia memory usage

### PR DESCRIPTION
**Pull request by ipinak**
This PR adds a counter metric to show the accumulated memory usage of mnesia tables.

### CODE REVIEW

**Comment by essen** [comment]
The metric name should include the unit, for example `erlang_mnesia_memory_usage_words` or `erlang_mnesia_memory_usage_bytes`.

There is `erlang_vm_wordsize_bytes` https://github.com/deadtrickster/prometheus.erl/commit/fd2a750e5dc8051459f773bf643a34847d0003e4 so perhaps you shouldn't do the words to bytes conversion when exporting and just do it in diagrams or other.

/cc @gerhard

**Comment by gerhard** [comment]
Agreed, `erlang_mnesia_memory_usage_bytes` makes most sense.

See the [Prometheus documentation on metric & label naming](https://prometheus.io/docs/practices/naming/) for context.

**Comment by ipinak** [comment]
> The metric name should include the unit, for example `erlang_mnesia_memory_usage_words` or `erlang_mnesia_memory_usage_bytes`.
> 
> There is `erlang_vm_wordsize_bytes` [fd2a750](https://github.com/deadtrickster/prometheus.erl/commit/fd2a750e5dc8051459f773bf643a34847d0003e4) so perhaps you shouldn't do the words to bytes conversion when exporting and just do it in diagrams or other.
> 
> /cc @gerhard

If I don't do the conversion to bytes, should the metric be named `erlang_mnesia_memory_usage_words` instead?

**Comment by gerhard** [comment]
Yes, if the value was no converted to bytes, then `_words` would have been the correct suffix.

The `word` unit however would make it harder to visualise it in e.g. Grafana, because there is no such data metric unit. `_bytes` is spot-on!

File: doc/prometheus_mnesia_collector.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ Number of transactions logged.
 Type: counter.<br />
 Total number of transaction restarts.
 
+* `erlang_mnesia_memory_usage_bytes`<br/>
+Type: counter.<br/>
```
Comment (gerhard) [inline doc/prometheus_mnesia_collector.md:54]: I believe this is a [gauge](https://prometheus.io/docs/concepts/metric_types/#gauge), not a [counter](https://prometheus.io/docs/concepts/metric_types/#counter). If you agree, can you please update it?

Happy to QA & merge afterwards 😉

Comment (ipinak) [reply doc/prometheus_mnesia_collector.md:54]: You are absolutely right, a gauge is more suitable.

**Pull request merged**